ScriptureRenderingPipelineWorker/RenderingTrigger.cs

```diff
@@ -136,7 +136,7 @@ private static async Task<RenderingResultMessage> RenderRepoAsync(WACSMessage me
 				    Message = "Unable to determine type of repo"
 			    };
 		    }
-			fileTracker = new FileTrackingLogger($"{rendererInput.BaseUrl}/u/{message.User}/{message.Repo}", repoType);
+			fileTracker = new FileTrackingLogger($"{rendererInput.BaseUrl.TrimEnd('/')}/u/{message.User}/{message.Repo}", repoType);
```

Comment:
This looks good.  It would be good to add a test at some point that confirms correct behavior.